src/include/platform/CHIPDeviceConfig.h

```diff
@@ -268,6 +268,19 @@
 #define CHIP_DEVICE_CONFIG_USER_SELECTED_MODE_TIMEOUT_SEC 30
 #endif // CHIP_DEVICE_CONFIG_USER_SELECTED_MODE_TIMEOUT_SEC
 
+/**
+ * CHIP_DEVICE_CONFIG_TEST_ROTATING_DEVICE_ID_UNIQUE_ID
```

Comment:
Should this go in the "standalone" configs, instead of being set for everyone by default?

Reply:
I believe that in the final protocol shape it should not be present as a config at all. That is because this unique ID should be programmed during factory provisioning. I think it is quite similar to the serial number value, so I based on that approach and defined test config value.

Reply:
Right, but the test config value should probably not get defined for all SDK consumers like we do here...

Reply:
To be honest I'm not sure what should I do with that one. Maybe giving a TEST name to this define was unfortunate, so I renamed it, so handling this value is the same, as for the other configs like: `CHIP_DEVICE_CONFIG_DEVICE_PRODUCT_NAME`, `CHIP_DEVICE_CONFIG_DEVICE_VENDOR_NAME` and etc